test/unit/dialects/oracledb.js

```diff
@@ -39,4 +36,56 @@ describe("OracleDb externalAuth", function() {
       expect(spy).to.have.callCount(1);
       expect(spy).to.have.been.calledWith(connectionWithExternalAuth);
   });
+
+  after(function() {
+    sinon.restore(knexInstance.client.driver.getConnection);
+  });
+
+});
+
+describe("OracleDb parameters", function() {
+  var knexClient;
+
+  before(function(done) {
+    var conf = _.clone(config.oracledb);
+    conf.fetchAsString = [ 'number', 'DATE', 'cLOb'];
+    knexClient = knex(conf);
+    knexClient.schema.createTable('fetchAsStringTable', function (table) {
+      table.increments();
+      table.float('testfloat');
+      table.date('testdate');
+    }).then(function() {
+      knexClient('fetchAsStringTable').insert({
+        testfloat: 7.32,
+        testdate: new Date(2017, 5, 7)
+      }).then(function() {
+        done();
```

Comment:
It would be better to just return promise from function instead of calling done in various places.